### ISSUE TITLE
correct source path

### ISSUE THREAD

**Issue by ralsal** [OlivierV78/switch_amd_nvidia#2]
Thank you for your work. Just installed it with Artix Linux. It installed in "/usr/local/bin".

First time I tried "switch_vers_nvidia" with my XMG core 15 E22, there was an error it could not find the file "10-amdgpu.conf_modif". It's obvious that you should either cd into the path of the installation. Or use full qualified path for the cp.

The following fix does the job on Artix Linux:
sudo cp /usr/local/bin/10-amdgpu.conf_modif /usr/share/X11/xorg.conf.d/10-amdgpu.conf
sudo cp /usr/local/bin/10-nvidia.conf_modif /usr/share/X11/xorg.conf.d/10-nvidia.conf

**Comment by OlivierV78** [comment]
Yes but for right work you will have to launch it from the install path.
With your bug fix it will only work if it is installer in /usr/local/bin/ but not in an other folder (I installed it in $HOME/switch_vers_nvidia

If I have time I will add the installation path in the script.
